airbyte-integrations/connectors/source-jira/metadata.yaml

```diff
@@ -30,6 +30,12 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      2.0.0:
```

Comment:
The source Jira connector is being migrated from the Python CDK to our declarative low-code CDK. Due to changes to the incremental stream state, this migration constitutes a breaking change. Additionally, an issue in the `ProjectAvatars` stream has been fixed. After updating, please reset your source before resuming syncs. For more information, see our migration documentation for source Jira.

Reply:
done